stockflux-launcher/package.json

```diff
@@ -20,7 +20,7 @@
     "throat": "4.1.0"
   },
   "scripts": {
-    "start": "cross-env BROWSER=none PORT=8050 react-scripts start",
+    "start": "cross-env BROWSER=none PORT=8055 react-scripts start",
```

Comment:
The chart container is running on this port, 8053 and 8054 are free

Reply:
Oh, sure, will change it to another port then. Thanks! Fixed -> 0856a64f169955e9de80e8a10b4d0db4d7833678